mollie/api/resources/customer_subscriptions.py

```diff
@@ -8,15 +8,13 @@ class CustomerSubscriptions(Base):
     customer_id = None
 
     def get_resource_object(self, result):
-        subscription = Subscription(result)
-        subscription._resource = self
-        return subscription
+        return Subscription(result, self)
 
     def get(self, subscription_id, **params):
         if not subscription_id or not subscription_id.startswith(self.RESOURCE_ID_PREFIX):
             raise IdentifierError(
-                'Invalid subscription ID: "%s". A subscription ID should start with "%s".' % (
-                    subscription_id, self.RESOURCE_ID_PREFIX)
+                "Invalid subscription ID: '{id}'. A subscription ID should start with '{prefix}'.".format(
```

Comment:
I'd prefer the `.format()` on a new line.

Reply:
We currently do this everywhere in the code: if we need to wrap code (>119 chars) and we can do it on an open parenthese, we use that. Changing that in a few places is more disturbing IMHO.

Reply:
Alright, fair.